packages/terra-application-utility/src/utility/_UtilityMenu.jsx

```diff
@@ -199,6 +199,7 @@ class UtilityMenu extends React.Component {
     const { childKeys } = this.getItem(key);
     const item = this.getItem(key);
     if (childKeys && childKeys.length > 0) {
+      // TODO: Figure out intention of previousKey variable
```

Comment:
The linter throws an error saying `previousKey` is unused state. Looking in this file, it doesn't seem to be referenced elsewhere. @yuderekyu do you have any thoughts on what this was intended for? There are refernces for `previousKeyStack`.

Reply:
Refactored this with the help of @tbiethman and @yuderekyu: https://github.com/cerner/terra-framework/pull/500/commits/dbd7a710b5ace0e20de2824186b3e885aa2647f2